deploy/k8s/minio.yml.j2

```diff
@@ -11,7 +11,7 @@ metadata:
 spec:
   containers:
     - name: minio-main
-      image: faasm/minio:0.8.12
+      image: faasm/minio:{{ faasm_version }}
```

Comment:
What other contents are we templating? I had thought I'd removed all templating from the repo as it makes it difficult to review changes, and generally adds more complexity. With this change we're now mixing SGX and non-SGX files again which I'd also quite like to avoid (just to keep the standard non-SGX setup as simple as possible).